hooks/pre-command

```diff
@@ -29,15 +23,17 @@ image_tag="$(compute_tag ${docker_file})"
 
 if ! docker pull "${image_name}:${image_tag}"; then
   echo "Image not cached, building"
+  docker build . --file "${docker_file}" -t "${image_name}:${image_tag}" || exit 1
   if [ "$BUILDKITE_BRANCH" == "$BUILDKITE_PIPELINE_DEFAULT_BRANCH" ]; then
-    docker build . --file "${docker_file}" -t "${image_name}:${image_tag}" -t "${image_name}:latest" || exit 1
-  else
-    docker build . --file "${docker_file}" -t "${image_name}:${image_tag}" || exit 1
+    docker tag "${image_name}" "latest"
+    docker push "${image_name}" "latest"
   fi
 fi || echo "Not found"
-  
-docker tag "${image_name}" "branch-$BUILDKITE_BUILD_NUMBER"
-docker push "${image_name}"
+
+for tag in ${BUILDKITE_PLUGIN_DOCKER_ECR_PUBLISH_ADDITIONAL_TAGS}
```

Comment:
Is `BUILDKITE_PLUGIN_DOCKER_ECR_PUBLISH_ADDITIONAL_TAGS ` a Bash array? Other plugins seem to require complicated parsing: https://github.com/buildkite-plugins/docker-compose-buildkite-plugin/blob/c1585516d948a71531540850cb78ade47ca35f53/lib/shared.bash#L61

I believe recommended syntax is

```shell
for element in "${array[@]}"
do
# something
done
```